src/components/Icon/Icon.stories.tsx

```diff
@@ -57,4 +57,28 @@ CustomIcons.args = {
 };
 CustomIcons.argTypes = {
   name: getCustomIcons(),
+  color: {
+    table: {
+      disable: true,
+    },
+  },
+  filled: {
+    table: {
+      disable: true,
+    },
+  },
+  disabled: {
+    table: {
+      disable: true,
+    },
+  },
+};
```

Comment:
You can use buildExcludeArgTypes(['color', 'filled', 'disabled']) to save a bit of space :)